README.md

```diff
@@ -157,12 +157,6 @@ $ grunt bump:prerelease
 >> Tagged as "v1.0.0-1"
 >> Pushed to origin
 
-$ grunt bump:patch
```

Comment:
"Removes a potentially confusing and redundant example in README.md"

bump:patch is already covered quite nicely above. I spent a couple minutes trying to figure out what that part of the README was explaining and all I could come up with is that it was a duplicate of the example for bump:patch above.